src/components/Navbar.tsx

```diff
@@ -58,36 +60,36 @@ const parentItems: Array<ParentItem> = [
         description: 'Get a rundown on all our projects',
         iconSrc: Map,
       },
-      {
-        name: 'Planner',
-        link: 'https://planner.utdnebula.com/',
-        description: 'Help plan degree and course requirements',
-        iconSrc: Pencil,
-      },
       {
         name: 'Jupiter',
-        link: 'https://jupiter.utdnebula.com/',
+        link: '/projects/jupiter',
         description: 'Find and connect with student organizations',
         iconSrc: Users,
       },
       {
         name: 'Trends',
-        link: 'https://trends.utdnebula.com/',
+        link: '/projects/trends',
         description: 'Help plan coursework through grade and professor stats',
         iconSrc: TrendingUp,
       },
       {
         name: 'Skedge',
-        link: 'https://chromewebstore.google.com/detail/skedge/ghipfanpcodcmkjacmmfjdmccdiaahab',
+        link: '/projects/skedge',
         description: 'Integrate grade and professor stats into Schedule Planner',
         iconSrc: TrendingUp,
       },
       {
         name: 'API & Platform',
-        link: 'https://github.com/UTDNebula/nebula-api',
+        link: '/projects/api',
         description: 'Integrate X+ years of historical UTD data into your applications',
```

Comment:
We can open a new issue for this but should probably have an actual number instead of an x.

Reply:
I asked Josh, pretty sure it's since 2017